Cannot build `updater` tool.

While following the instructions for compiling the updater tool, I got stuck here:

1. Ran `nix-shell`
2. Ran `npm run build`
3. Got this error: `sh: ncc: command not found`:
```
[nix-shell:~/Programming/Git Projects/purescript-contrib-governance/updater]$ npm run build

> contrib-updater@ build /home/jordan/Programming/Git Projects/purescript-contrib-governance/updater
> spago bundle-module --to output/index.js && ncc build --minify --out bin index.js

[info] Installation complete.
           Src   Lib   All
Warnings   0     0     0  
Errors     0     0     0  
[info] Build succeeded.
[info] Bundling first...
[info] Bundle succeeded and output file to output/index.js
[info] Make module succeeded and output file to output/index.js
sh: ncc: command not found
npm ERR! code ELIFECYCLE
npm ERR! syscall spawn
npm ERR! file sh
npm ERR! errno ENOENT
npm ERR! contrib-updater@ build: `spago bundle-module --to output/index.js && ncc build --minify --out bin index.js`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the contrib-updater@ build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/jordan/.npm/_logs/2020-08-27T20_42_15_055Z-debug.log
```

--- a/updater/README.md
+++ b/updater/README.md
@@ -6,26 +6,17 @@ Please see the [contrib-updater documentation](./docs) to learn about the comman
 
 ## Installation
 
-First, ensure you have all necessary dependencies by entering a developer shell:
+The Nix shell provides all necessary dependencies and hooks for installing dependencies and building the tool. Start by entering the shell:
 
 ```sh
 nix-shell
 ```
 
-Then, build the executable by running the `build` script:
+You can now use `contrib-updater` to run the tool:
 
 ```sh
-npm run build
+contrib-updater --help
 ```
-
-Finally, run `npm link` to add the executable to your PATH:
-
-```sh
-# Nix restricts permissions, so sudo is required if inside the shell.
-sudo npm link
-```
-
-You can now use the `contrib-updater` executable.
 
 ## Usage
 

--- a/updater/shell.nix
+++ b/updater/shell.nix
@@ -19,4 +19,10 @@ in pkgs.stdenv.mkDerivation {
     pkgs.nodejs-12_x
     pkgs.dhall-json
   ];
+  executable = ./bin/index.js;
+  shellHook =''
+    npm install
+    npm run build
+    alias contrib-updater='${pkgs.nodejs-12_x}/bin/node ${./.}/bin/index.js'
+  '' ;
 }

